cpp/velox/compute/VeloxBackend.cc

```diff
@@ -138,10 +138,14 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
     LOG(INFO) << "VeloxBackend config:" << printConfig(veloxcfg->valuesCopy());
   }
 
-  uint32_t vlogLevel = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
-  uint32_t severityLogLevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
-  FLAGS_v = vlogLevel;
-  FLAGS_minloglevel = severityLogLevel;
+  if (!veloxcfg->get<bool>(kDebugModeEnabled, false)) {
+    uint32_t vlogLevel = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
+    FLAGS_v = vlogLevel;
+    uint32_t severityLogLevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
+    FLAGS_minloglevel = severityLogLevel;
+  } else {
+    FLAGS_v = 99;
```

Comment:
Is there one case that user set `kDebugModeEnabled` to `true` then tended to set glog level to lower value? With the change user would not be able to change glog level in debug mode.

Reply:
Do you mean `FLAGS_v=99` is strange? if yes, I can revert, verbose log is different with severity log, I just open them all for convenience.

Most debug info be printed by LOG(INFO), I can't image someone enable debug mode but disable log print.

Reply:
> Do you mean FLAGS_v=99 is strange? 

This is OK to me. I am feeling that we have a couple of different options to control logging. Probably it's necessary to do some sort of simplifications.